api/src/main/kotlin/edu/wgu/osmt/RoutePaths.kt

```diff
@@ -2,7 +2,9 @@ package edu.wgu.osmt
 
 object RoutePaths {
     const val SEARCH_PATH = "/api/search"
+    const val SCROLL_PATH = "/api/scroll"
```

Comment:
"scroll" is an implementation detail about ES. In this context, we want to express the REST API. That's more about "export" than "scroll". For now, I recommend that we remove SCROLL_PATH, and use "api/export/library" for this endpoint. It's kind of a one-off special admin case.

Reply:
nice, will do, thanks!